src/index.js

```diff
@@ -19,11 +19,20 @@ import { config } from './utils/oktaConfig';
 import { LoadingComponent } from './components/common';
 import { WorkOrderPage } from './components/pages/WorkOrderForm';
 
+import { createStore } from 'redux';
+import allReducers from './state/reducers';
+import { Provider } from 'react-redux';
+
+const store = createStore(
+    allReducers, window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__()
```

Comment:
I'm a bit unfamiliar with Redux (it's been a while 😅 ), this looks fine but I'm just curious about the purpose of including references to the Redux Devtools Extension.

Reply:
Haha I included the extension for so many reasons so when you pull up the Redux dev tools in the browser, you can for instance i used it to see if the counter buttons were working and if 'isLogged' was true or false. It gives the ability to rewind state and see what the app was doing everytime state changed.